src/flint/fmpz_mat.jl

```diff
@@ -476,6 +476,14 @@ end
 
 ==(x::ZZRingElem, y::ZZMatrix) = parent(y)(x) == y
 
+# Return a positive integer if A[i, j] > b, negative if A[i, j] < b, 0 otherwise
+function compare_index(A::ZZMatrix, i::Int, j::Int, b::ZZRingElem)
```

Comment:
This name is bad (we are not comparing indices). Perhaps we should move it back to Hecke, then it can be renamed there, and in fact reimplemented using `mat_entry_ptr` plus `cmp`.